### DISCUSSION TITLE
LFX Mentorship '24 Term 3 Sept - Nov: Open for Project Proposals

### DISCUSSION THREAD

**Discussion by nate-double-u** [cncf/mentoring#1263] [Announcements]
Hello everyone!

The LFX Mentorship 2024 Term 3 Sept - Nov is open now open for project ideas:
https://github.com/cncf/mentoring/blob/main/programs/lfx-mentorship/2024/03-Sep-Nov

Project submission and application timeline:
* project proposal period: **Jul 3 - July 30, 2024**
* mentee applications open: **Jul 31 - Aug 13**
* application review/admission decisions/HR paperwork: **Aug 14 - Aug 27**

We're looking forward to seeing all the project ideas you're interested in working on!

### _Updated_ Timeline

| activity                                                                                                     | date                                                                                |
|--------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------|   
| project proposals                                                                                            | Jul 3 - **Jul 30** 5:00 PM PDT                                                          |
| mentee applications open                                                                                     | **Jul 31 - Aug 13** 5:00 PM PDT                                                         |
| application review/admission decisions                                                                       | **Aug 14 - Aug 27** 5:00 PM PDT                                                         |
| selection notifications                                                                                      | **Sept 3**, 2024 5:00 PM PDT                                                            |
| Mentorship program begins with the initial work assignments                                                  | **Sept 9**, 2024                                                                        | 
| Midterm mentee evaluations and first stipend payments                                                        | October 15, 2024 5:00 PM PDT (the week following the National Day holiday in China) |
| Final mentee evaluations and mentee feedback/blog submission due, second and final stipend payment approvals | Nov 26, 2024 5:00 PM PDT (Giving some time after KubeCon NA)                        |
| Last day of term                                                                                             | November 29, 2024                                                                   |

**Comment by Ankit-kumar-rajput** [comment]
where is project ideas? @nate-double-u

**Reply by upsaurav12** [reply]
https://github.com/cncf/mentoring/blob/main/programs/lfx-mentorship/2024/03-Sep-Nov/project_ideas.md
Here is the link where all the ``CNCF`` project proposals will be listed

**Reply by upsaurav12** [reply]
Projects are not listed yet can you tell me exactly when will projects listed??

**Reply by nate-double-u** [reply]
We invite the projects to make proposals, they do it on their time. The goal is to have them in by July 23, but we may need to extend that if we don't get many in by then.

**Reply by upsaurav12** [reply]
thanks for the reply

**Comment by nate-double-u** [comment]
We've extended the project proposal period!

I've updated the info above, highlighting the new dates in bold. Thanks for your patience everyone!

**Reply by that-avg-dev** [reply]
@nate-double-u can you guide me on how to contribute by the time applications are due to the org? Regarding LFX only

**Reply by aliok** [reply]
@kairveeehh 

Timeline is available here: https://github.com/cncf/mentoring/tree/main/programs/lfx-mentorship/2024/03-Sep-Nov

To get started with contributions, you can have a look at https://contribute.cncf.io/contributors/getting-started/

**Reply by saashaspace** [reply]
@nate-double-u the applications shall be live by tomorrow on the LFX mentorship website right?

**Reply by lazyperson1020** [reply]
@nate-double-u Can I apply to more than one project in the SAME organization for Sep-Nov term?

**Reply by nate-double-u** [reply]
> @nate-double-u Can I apply to more than one project in the SAME organization for Sep-Nov term?

Hi @lazyperson1020,

This program is very competitive, sometimes garnering hundreds of applications per project proposal, so I'd suggest putting your best foot forward. Stating that you're lazy and then asking a question that is easily searchable doesn't do that. Setting the question aside for a moment, setting your public persona as a "lazy person" isn't a good strategy—I can see from your GitHub profile that you aren't lazy; you've clearly done work—you set in your audience's mind that they're dealing with someone who may be lazy, which when you're trying to convince someone to hire you, works against you.

**Reply by lazyperson1020** [reply]
> > @nate-double-u Can I apply to more than one project in the SAME organization for Sep-Nov term?
> 
> Hi @lazyperson1020,
> 
> This program is very competitive, sometimes garnering hundreds of applications per project proposal, so I'd suggest putting your best foot forward. Stating that you're lazy and then asking a question that is easily searchable doesn't do that. Setting the question aside for a moment, setting your public persona as a "lazy person" isn't a good strategy—I can see from your GitHub profile that you aren't lazy; you've clearly done work—you set in your audience's mind that they're dealing with someone who may be lazy, which when you're trying to convince someone to hire you, works against you.

Dearest @nate-double-u , thanks for your advice and insights. I hadn’t considered the implications of my username choice, which was completely random, and I will change it in some time to better reflect my professionalism. I hope the project maintainers will also focus on evaluating my work rather than the impression my username might give. As for my question, I was unclear about whether can we apply  for two different projects within the same organization and whether it’s appropriate enough to do so. Anyways, Thank you again for your time and efforts.
